src/list.scss

```diff
@@ -103,15 +106,22 @@ $block: #{$fd-namespace}-list;
     }
   }
 
+  &__form-item {
```

Comment:
The base `fd-form-item` does not have properties margin-top and margin-left that `fd-list__form-item` is overwriting so there's no chance of conflict where we will depend on the order of import. This is an issue when the prop exists in the base class and we overwrite it, like the case with the background color. But anyways, doesn't hurt to change it so I did it :)

Reply:
It now has both: 
```
.fd-list__form-item,
.fd-list__form-item.fd-list-item  {...}
```

since we need to test the selection and we cannot use `fd-list-item` inside the List test.